src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Commands.AzureStackStorage.csproj

```diff
@@ -54,6 +54,13 @@
     <Reference Include="Microsoft.Azure.Common">
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Azure.Common.Authentication">
```

Comment:
Please update to the latest - this should not be necessary, you should have a project dependency on commands.common.authentication instead